server/src/main/java/org/elasticsearch/transport/TransportStats.java

```diff
@@ -94,6 +121,29 @@ public ByteSizeValue getTxSize() {
         return txSize();
     }
 
+    public long[] getHandlingTimeBucketFrequencies() {
+        final long[] histogram = new long[handlingTimeBucketFrequencies.length];
+        System.arraycopy(handlingTimeBucketFrequencies, 0, histogram, 0, handlingTimeBucketFrequencies.length);
+        return histogram;
+    }
+
+    public int[] getHandlingTimeBucketBounds() {
+        final int[] bounds = new int[handlingTimeBucketBounds.length];
+        System.arraycopy(handlingTimeBucketBounds, 0, bounds, 0, handlingTimeBucketBounds.length);
+        return bounds;
```

Comment:
NIT: I suppose you could use `Arrays.copyOf` or `clone` to shorten these to a single line :)